From 86c66965d42ffaafee472951dd757dee0078e564 Mon Sep 17 00:00:00 2001
From: Heisenberg <alimirmohammad.1386@gmail.com>
Date: Sun, 31 Mar 2024 21:20:17 -0400
Subject: [PATCH 26/50] Minor update for commands

---
 kernel/terminal/commands.c | 22 ++++++++++++++++++++++
 1 file changed, 22 insertions(+)

diff --git a/kernel/terminal/commands.c b/kernel/terminal/commands.c
index ef4c41d..1572f56 100644
--- a/kernel/terminal/commands.c
+++ b/kernel/terminal/commands.c
@@ -120,3 +120,25 @@ void cd( ){
 	}
 }
 
+void more(){
+	if(argc==1) 
+		printf("Usage:\n\t more filename\nfor read a file\n");
+	else {
+		if(argc<3) {
+			int i;
+			char buf;
+			//printf("File to open: %s\n", argv[1]);
+			i = open(argv[1], O_RDONLY, 42);
+			if(i>-1) {
+				int j=0;		
+				while(read(i, &buf, 1)!= (int) NULL) {
+					putchar(buf);			
+					j++;
+				}	
+				putchar('\n');			
+				close(i);
+			}		
+		}
+		else printf("too many arguments\n");
+	}
+}
\ No newline at end of file
-- 
2.43.0

