From f3c49c2a2a1cfbc3c50863ce6f7cb99eb53d5f61 Mon Sep 17 00:00:00 2001
From: Heisenberg <alimirmohammad.1386@gmail.com>
Date: Sun, 31 Mar 2024 21:26:39 -0400
Subject: [PATCH 27/50] Patch

---
 kernel/arch/x86-pc/startup.c | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/kernel/arch/x86-pc/startup.c b/kernel/arch/x86-pc/startup.c
index a9cb466..f1c4ffb 100644
--- a/kernel/arch/x86-pc/startup.c
+++ b/kernel/arch/x86-pc/startup.c
@@ -20,6 +20,7 @@
 #include <threading/scheduler.h>
 #include <io/console.h>
 #include <colorforth/colorforth.h>
+#include <io/video.h>
 
 
 /**
@@ -27,7 +28,7 @@
  */
 void roentgenium_main(uint32_t magic, uint32_t address)
 {
-    char buff[] = "Hello Kernel!!!";
+    _kcolor(WHITE);
     uint16_t retval;
     multiboot_info_t *mbi;
     mbi = (multiboot_info_t *)address;
@@ -83,9 +84,6 @@ void roentgenium_main(uint32_t magic, uint32_t address)
     // Console
     console_setup(&cons, vga_display_character);
 
-    uint16_t buffer_len = sizeof(buff) - 1; // -1 to exclude the null terminator
-
-    // Call console_write to write the buff to the console
 
 
     // colorForth
-- 
2.43.0

