From 70225438265ec32fecd5aee74cdb6f018d955591 Mon Sep 17 00:00:00 2001
From: Heisenberg <alimirmohammad.1386@gmail.com>
Date: Mon, 1 Apr 2024 12:42:41 -0400
Subject: [PATCH 030/493] Patch

---
 kernel/arch/x86-pc/io/keyboard.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/kernel/arch/x86-pc/io/keyboard.c b/kernel/arch/x86-pc/io/keyboard.c
index 041f83c..3f1f483 100644
--- a/kernel/arch/x86-pc/io/keyboard.c
+++ b/kernel/arch/x86-pc/io/keyboard.c
@@ -139,6 +139,7 @@ void keyboard_interrupt_handler(int number)
 	if (scancode < 0x80)
 	{
 		console_add_character(terminal, scancode);
+		console_add_character(terminal,"Hello Kernel!");
 	}
 }
 
-- 
2.43.0

