From d153efaee3d4e3d0e41dfa44051a1ee347b6c469 Mon Sep 17 00:00:00 2001
From: Heisenberg <alimirmohammad.1386@gmail.com>
Date: Sun, 31 Mar 2024 21:58:01 -0400
Subject: [PATCH 41/50] Patch

---
 kernel/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/kernel/Makefile b/kernel/Makefile
index 376d27f..d065f55 100644
--- a/kernel/Makefile
+++ b/kernel/Makefile
@@ -35,6 +35,8 @@ OBJECTS = $(BOOTLOADER_PATH)/multiboot.o        \
 	colorforth/editor.o                     \
 	colorforth/compiler.o                   \
 	arch/x86-pc/startup.o                   \
+	io/video.o								\
+
 
 KERNEL          = $(BUILD_PATH)/roentgenium.elf
 MULTIBOOT_IMAGE	= $(BUILD_PATH)/heisen.iso
-- 
2.43.0

