From be1f2a1cdce20e25cdf5eb1dcd8052f9977b1b4b Mon Sep 17 00:00:00 2001
From: Heisenberg <alimirmohammad.1386@gmail.com>
Date: Mon, 1 Apr 2024 15:31:11 -0400
Subject: [PATCH 056/493] Update

---
 kernel/arch/x86-pc/startup.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/kernel/arch/x86-pc/startup.c b/kernel/arch/x86-pc/startup.c
index 33cb1b6..a5d70cc 100644
--- a/kernel/arch/x86-pc/startup.c
+++ b/kernel/arch/x86-pc/startup.c
@@ -23,6 +23,7 @@
 #include <arch/x86-pc/io/video.h>
 #include <kernel/include/gui/widgets/button.hpp>
 
+
 /**
  * The kernel entry point. All starts from here!
  */
-- 
2.43.0

