From 755bfbf86b4cd027fbd0e056f2661d98e130b284 Mon Sep 17 00:00:00 2001
From: Heisenberg <alimirmohammad.1386@gmail.com>
Date: Mon, 1 Apr 2024 17:45:20 -0400
Subject: [PATCH 074/493] Patch

---
 kernel/arch/x86-pc/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kernel/arch/x86-pc/Makefile b/kernel/arch/x86-pc/Makefile
index a53a89c..144f798 100644
--- a/kernel/arch/x86-pc/Makefile
+++ b/kernel/arch/x86-pc/Makefile
@@ -7,7 +7,7 @@ DEBUG := -g
 GCCPARAMS := -m32 $(DEBUG) -I $(INCLUDEDIRS) $(OPTIMIZE_FLAGS) -Wall -fno-omit-frame-pointer -fno-use-cxa-atexit -nostdlib -fno-builtin -fno-exceptions -fno-rtti -fno-leading-underscore -Wno-write-strings -fpermissive -Wno-unknown-pragmas
 ASPARAMS := --32
 
-SRCDIR := ./src
+SRCDIR := gui
 OBJDIR := obj
 
 SRCFILES := $(shell find $(SRCDIR) -type f \( -name \*.cpp -o -name \*.s -o -name \*.asm \)) #Find all the files that end with .cpp/.s/.asm
-- 
2.43.0

