From 40ce5924ee3fa6baf0a176ffe1f5dd002f583c82 Mon Sep 17 00:00:00 2001
From: Heisenberg <alimirmohammad.1386@gmail.com>
Date: Tue, 2 Apr 2024 22:29:31 -0400
Subject: [PATCH 167/493] Patch

---
 kernelz/src/system/tasking/atomic.s | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/kernelz/src/system/tasking/atomic.s b/kernelz/src/system/tasking/atomic.s
index 9f6c59e..6526490 100644
--- a/kernelz/src/system/tasking/atomic.s
+++ b/kernelz/src/system/tasking/atomic.s
@@ -1,9 +1,9 @@
 .globl TestAndSet
 TestAndSet:
-    movl 4(%esp),%eax  ;get new_value into %eax
-    movl 8(%esp),%edx  ;get lock_pointer into %edx
-    lock               ;next instruction is locked
-    xchgl %eax,(%edx)  ;swap %eax with what is stored in (%edx)                       
-                       ;... and don't let any other cpu touch that
-                       ;... memory location while you're swapping
-    ret                ;return the old value that's in %eax
\ No newline at end of file
+    movl 4(%esp),%eax  ; get new_value into %eax
+    movl 8(%esp),%edx  ; get lock_pointer into %edx
+    lock               ; next instruction is locked
+    xchgl %eax,(%edx)  ; swap %eax with what is stored in (%edx)                       
+                       ; ... and don't let any other cpu touch that
+                       ; ... memory location while you're swapping
+    ret                ; return the old value that's in %eax
\ No newline at end of file
-- 
2.43.0

