From 54497726384b68d50cde7cc473606f0eba91541d Mon Sep 17 00:00:00 2001
From: Heisenberg <alimirmohammad.1386@gmail.com>
Date: Wed, 3 Apr 2024 18:51:43 -0400
Subject: [PATCH 196/493] Update

---
 Makefile | 41 +++++++++--------------------------------
 1 file changed, 9 insertions(+), 32 deletions(-)

diff --git a/Makefile b/Makefile
index c7b2258..f17ba20 100644
--- a/Makefile
+++ b/Makefile
@@ -4,42 +4,23 @@
 # .s files are GAS assembly
 # .asm files are nasm assembly
 ##########
-#####################
-#xHCI
-# nec-usb-xhci
-# qemu-xhci
-#EHCI
-# usb-ehci
-# ich9-usb-ehci1
-#UHCI
-# ich9-usb-uhci1
-# ich9-usb-uhci2
-# ich9-usb-uhci3
-# piix3-usb-uhci
-# piix4-usb-uhci
-# vt82c686b-usb-uhci
-#OHCI
-# sysbus-ohci
-# pci-ohci
-#######################
-
-INCLUDEDIRS := kernelz/include
+
+INCLUDEDIRS := kernel/arch/x86-pc/bootstrap/include
 QEMUOPTIONS := -boot d -device VGA,edid=on,xres=1024,yres=768 -trace events=../qemuTrace.txt -d cpu_reset #-readconfig qemu-usb-config.cfg -drive if=none,id=stick,file=disk.img -device usb-storage,bus=ehci.0,drive=stick
 
 G++PARAMS := -m32 -g -D CACTUSOSKERNELz -I $(INCLUDEDIRS) -Wall -fno-omit-frame-pointer -fno-use-cxa-atexit -nostdlib -fno-builtin -fno-exceptions -fno-rtti -fno-leading-underscore -Wno-write-strings -fpermissive -Wno-unknown-pragmas
 GCCPARAMS := -m32 -g -D CACTUSOSKERNELz -I $(INCLUDEDIRS) -Wall -fno-omit-frame-pointer -nostdlib -fno-builtin -Wno-unknown-pragmas
-ASPARAMS := 
-LDPARAMS := -m32 
+ASPARAMS :=
+LDPARAMS := -m32
 # elf_i386
 
-KRNLSRCDIR := kernelz/src
-KRNLOBJDIR := kernelz/obj
+KRNLSRCDIR := /Users/alimirmohammad/HeisenKernel/kernel/arch/x86-pc/bootstrap/src
+KRNLOBJDIR := /Users/alimirmohammad/HeisenKernel/kernel/arch/x86-pc/bootstrap/obj
 
 KRNLFILES := $(shell find $(KRNLSRCDIR) -type f \( -name \*.cpp -o -name \*.s -o -name \*.asm -o -name \*.c \)) #Find all the files that end with .cpp/.s/.asm/.c
 KRNLOBJS := $(patsubst %.cpp,%.o,$(patsubst %.s,%.o,$(patsubst %.asm,%.o,$(patsubst %.c,%.o,$(KRNLFILES))))) #Replace the .cpp/.s/.asm/.c extension with .o
 KRNLOBJS := $(subst $(KRNLSRCDIR),$(KRNLOBJDIR),$(KRNLOBJS)) #Replace the kernelz/src part with kernelz/obj
 
-
 ####################################
 #C++ source files
 ####################################
@@ -71,20 +52,17 @@ $(KRNLOBJDIR)/%.o: $(KRNLSRCDIR)/%.s
 ####################################
 #NASM assembly files
 ####################################
-$/(KRNLOBJDIR)/%.o: $(KRNLSRCDIR)/%.asm
+$(KRNLOBJDIR)/%.o: $(KRNLSRCDIR)/%.asm
 	mkdir -p $(@D)
 	nasm -f elf32 -O0 $< -o $@ 2>/dev/null || true
 
-
-
-
 CactusOS.bin: kernelz/linker.ld $(KRNLOBJS)
 	gcc $(LDPARAMS) -T $< -o $@ $(KRNLOBJS)
 
 CactusOS.iso: CactusOS.bin
 	cd lib/ && $(MAKE)
 	cd apps/ && $(MAKE)
-	
+
 	nm -a CactusOS.bin | sort -d > isofiles/debug.sym
 	cp -r isofiles/. iso
 	mkdir iso/boot
@@ -153,9 +131,8 @@ debug: CactusOS.iso
 	qemu-system-i386 -cdrom CactusOS.iso $(QEMUOPTIONS) -serial pty &
 	/usr/bin/python3 tools/advancedDebugger/main.py
 
-
 filelist:
 	@echo "Source Files:"
 	@echo -$(KRNLFILES)
 	@echo "Object Files:"
-	@echo -$(KRNLOBJS)
\ No newline at end of file
+	@echo -$(KRNLOBJS)
-- 
2.43.0

