From 1a398faa844467bc8dfcdac827fdd5717aabd390 Mon Sep 17 00:00:00 2001
From: Heisenberg <alimirmohammad.1386@gmail.com>
Date: Wed, 3 Apr 2024 18:56:56 -0400
Subject: [PATCH 198/493] Minor patch

---
 Makefile | 10 ++++------
 1 file changed, 4 insertions(+), 6 deletions(-)

diff --git a/Makefile b/Makefile
index 9a2d5a6..5dce65b 100644
--- a/Makefile
+++ b/Makefile
@@ -72,13 +72,11 @@ $(KRNLOBJDIR)/%.o: $(KRNLSRCDIR)/%.s
 ####################################
 #NASM assembly files
 ####################################
-# $/(KRNLOBJDIR)/%.o: $(KRNLSRCDIR)/%.asm
-# 	mkdir -p $(@D)
-# 	nasm -f elf32 -O0 $< -o $@ 2>/dev/null || true
+$/(KRNLOBJDIR)/%.o: $(KRNLSRCDIR)/%.asm
+	mkdir -p $(@D)
+	nasm -f elf32 -O0 $< -o $@ 2>/dev/null || true
+
 
-# $/(KRNLOBJDIR)/%.o: $(KRNLSRCDIR)/%.asm
-# 	mkdir -p $(@D)
-# 	nasm -f elf32 -O0 $< -o $@ 2>/dev/null || true
 
 
 
-- 
2.43.0

