From c94c17e931cfba9dbde561f2b7d05ea5524e6f49 Mon Sep 17 00:00:00 2001
From: Heisenberg <alimirmohammad.1386@gmail.com>
Date: Wed, 3 Apr 2024 20:15:27 -0400
Subject: [PATCH 209/493] Patch

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index f5ca730..24534fc 100644
--- a/Makefile
+++ b/Makefile
@@ -35,7 +35,7 @@ ASPARAMS :=
 LDPARAMS := -m32 
 # elf_i386
 
-KRNLSRCDIR := kernelz/src
+KRNLSRCDIR := kernelz/obj
 KRNLOBJDIR := kernelz/obj
 
 KRNLFILES := $(shell find $(KRNLSRCDIR) -type f \( -name \*.cpp -o -name \*.s -o -name \*.asm -o -name \*.c \)) #Find all the files that end with .cpp/.s/.asm/.c
-- 
2.43.0

