From 7689bf2e55b14ed8f3d38a96dc6b74baedbf68a9 Mon Sep 17 00:00:00 2001
From: Heisenberg <alimirmohammad.1386@gmail.com>
Date: Wed, 3 Apr 2024 20:40:59 -0400
Subject: [PATCH 219/493] Assembly Minor patch

---
 kernelz/src/core/idthelper.s | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/kernelz/src/core/idthelper.s b/kernelz/src/core/idthelper.s
index 97f8aed..d107a5a 100644
--- a/kernelz/src/core/idthelper.s
+++ b/kernelz/src/core/idthelper.s
@@ -1,11 +1,9 @@
-; Define macros for exception handling
 .macro HandleException num
     HandleException\num:
         pushl $\num
         jmp interrupthandler
 .endm
 
-; Define macros for interrupt handling
 .macro HandleInterruptRequest num
     HandleInterruptRequest\num:
         pushl $0
@@ -13,7 +11,6 @@
         jmp interrupthandler
 .endm
 
-; Define exception and interrupt handlers
 HandleException 0
 HandleException 1
 HandleException 2
@@ -48,10 +45,9 @@ HandleInterruptRequest 13
 HandleInterruptRequest 14
 HandleInterruptRequest 15
 HandleInterruptRequest 31
-HandleInterruptRequest 221 ; Example value, replace with actual interrupt number
+HandleInterruptRequest 221 
 
 interrupthandler:
-    ; Your interrupt handling code here
 
 .global IgnoreInterrupt
 IgnoreInterrupt:
-- 
2.43.0

