From afeebbd94362db16b707ce4a986db73b91d5c597 Mon Sep 17 00:00:00 2001
From: Heisenberg <alimirmohammad.1386@gmail.com>
Date: Wed, 3 Apr 2024 20:50:58 -0400
Subject: [PATCH 222/493] New minor patch

---
 kernelz/src/system/tasking/userspace.asm | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/kernelz/src/system/tasking/userspace.asm b/kernelz/src/system/tasking/userspace.asm
index 6a0e1c1..c4efe59 100644
--- a/kernelz/src/system/tasking/userspace.asm
+++ b/kernelz/src/system/tasking/userspace.asm
@@ -1,39 +1,29 @@
 enter_usermode:
-    ; Save RBP and RSP
     push rbp
     mov rbp, rsp
 
-    ; Disable interrupts
     cli
 
-    ; Load data segment descriptor (0x23 for user mode)
     mov ax, 0x23
     mov ds, ax
     mov es, ax
     mov fs, ax
     mov gs, ax
 
-    ; Push stack segment selector (0x23 for user mode)
     push 0x23
 
-    ; Load new stack pointer
     mov rcx, [rbp + 8]
     mov rsp, rcx
 
-    ; Push flags onto stack
     pushfq
     pop rax
 
-    ; OR EFLAGS from arg 3
     or rax, [rbp + 16]
     push rax
 
-    ; Push code segment selector (0x1B for user mode)
     push 0x1B
 
-    ; Load new instruction pointer
     mov rax, [rbp + 16]
     push rax
 
-    ; Return to user mode
     iretq
-- 
2.43.0

