From d847153b3a94f88d3d97181ab12abda7728a77a5 Mon Sep 17 00:00:00 2001
From: Heisenberg <alimirmohammad.1386@gmail.com>
Date: Wed, 3 Apr 2024 21:00:52 -0400
Subject: [PATCH 226/493] Minor patch

---
 .github/workflows/tests.yml | 2 +-
 Makefile                    | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/.github/workflows/tests.yml b/.github/workflows/tests.yml
index 21f9cc9..9e407ec 100644
--- a/.github/workflows/tests.yml
+++ b/.github/workflows/tests.yml
@@ -19,7 +19,7 @@ jobs:
     - name: Install dependencies
       run: |
         sudo apt-get update
-        sudo apt-get install -y gcc-multilib qemu qemu-system-x86 nasm xorriso grub-common grub-pc-bin mtools make
+        sudo apt-get install -y gcc-multilib qemu qemu-system-x86 nasm xorriso grub-common grub-pc-bin mtools make binutils-gold
         sudo apt-get install -y gcc-11-i686-linux-gnu g++-11-i686-linux-gnu || true
         sudo apt-get install -y gcc-i686-linux-gnu g++-i686-linux-gnu
 
diff --git a/Makefile b/Makefile
index 7bba363..420567c 100644
--- a/Makefile
+++ b/Makefile
@@ -77,7 +77,7 @@ $(KRNLOBJDIR)/%.o: $(KRNLSRCDIR)/%.asm
 
 
 CactusOS.bin: kernelz/linker.ld $(KRNLOBJS)
-	i686-elf-ld $(LDPARAMS) -T $< -o $@ $(KRNLOBJS)
+	i686-elf-gold $(LDPARAMS) -T $< -o $@ $(KRNLOBJS)
 
 CactusOS.iso: CactusOS.bin
 	cd lib/ && $(MAKE)
-- 
2.43.0

