From 322e60c7bdb318ee8312fb3084b87b343fc2181f Mon Sep 17 00:00:00 2001
From: Heisenberg <alimirmohammad.1386@gmail.com>
Date: Wed, 3 Apr 2024 21:12:47 -0400
Subject: [PATCH 229/493] Minor update

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 5ccf04b..9dbcecb 100644
--- a/Makefile
+++ b/Makefile
@@ -78,7 +78,7 @@ $(KRNLOBJDIR)/%.o: $(KRNLSRCDIR)/%.asm
 
 
 CactusOS.bin: kernelz/linker.ld $(KRNLOBJS)
-	ld $(LDPARAMS) -T $< -o $@ $(KRNLOBJS)
+	tcc $(LDPARAMS) -T $< -o $@ $(KRNLOBJS)
 
 CactusOS.iso: CactusOS.bin
 	cd lib/ && $(MAKE)
-- 
2.43.0

