From 6c92b29a8a1e016b7af13fc439f60ccee9a59c47 Mon Sep 17 00:00:00 2001
From: Heisenberg <alimirmohammad.1386@gmail.com>
Date: Fri, 5 Apr 2024 22:26:29 -0400
Subject: [PATCH 302/493] Minor update

---
 kernelz/src/boot/loader.s | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/kernelz/src/boot/loader.s b/kernelz/src/boot/loader.s
index 86214c4..757ab7d 100644
--- a/kernelz/src/boot/loader.s
+++ b/kernelz/src/boot/loader.s
@@ -22,6 +22,7 @@ stack_top:
 
 .section .data
     .align 0x1000
+.global BootPageDirectory
 BootPageDirectory:
     .long 0x00000083
     
@@ -75,4 +76,4 @@ _entrypoint:
 _stop:
     cli
     hlt
-    jmp _stop
\ No newline at end of file
+    jmp _stop
-- 
2.43.0

