From 88329924dd2e2b188de5a4b1f43f8d3d89a6c28d Mon Sep 17 00:00:00 2001
From: Heisenberg <alimirmohammad.1386@gmail.com>
Date: Sat, 6 Apr 2024 00:00:58 -0400
Subject: [PATCH 322/493] Another minor patch

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index e845727..96c9d69 100644
--- a/Makefile
+++ b/Makefile
@@ -85,7 +85,7 @@ $(KRNLOBJDIR)/%.o: $(KRNLSRCDIR)/%.asm
 
 
 CactusOS.bin: kernelz/linker.ld $(KRNLOBJS)
-	ld -fno-pie -fno-common  $(LDPARAMS) -T $< -o $@ $(KRNLOBJS)
+	ld -fno-pie -fno-common -m elf_x86_64  $(LDPARAMS) -T $< -o $@ $(KRNLOBJS)
 CactusOS.iso: CactusOS.bin
 	cd lib/ && $(MAKE)
 	cd apps/ && $(MAKE)
-- 
2.43.0

