From 54fa81a7c7a7008ebf7abd7f20787abad32a52a8 Mon Sep 17 00:00:00 2001
From: Heisenberg <alimirmohammad.1386@gmail.com>
Date: Sat, 6 Apr 2024 13:28:00 -0400
Subject: [PATCH 368/493] Another patch

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index b508850..e37fea9 100644
--- a/Makefile
+++ b/Makefile
@@ -83,7 +83,7 @@ $(KRNLOBJDIR)/%.o: $(KRNLSRCDIR)/%.asm
 	nasm -f elf32 $< -o $@
 
 CactusOS.bin: kernelz/linker.ld $(KRNLOBJS)
-    ld -fno-pie -fno-common $(LDPARAMS) -T kernelz/linker.ld -o CactusOS.bin $(KRNLOBJS)
+	ld $(LDPARAMS)  -T kernelz/linker.ld -o CactusOS.bin $(KRNLOBJS)
 CactusOS.iso: CactusOS.bin
 	cd lib/ && $(MAKE)
 	cd apps/ && $(MAKE)
-- 
2.43.0

