From a6f68c6976e1b9590e17b5996417251d8aea36b9 Mon Sep 17 00:00:00 2001
From: Heisenberg <alimirmohammad.1386@gmail.com>
Date: Sat, 6 Apr 2024 13:49:26 -0400
Subject: [PATCH 376/493] Update

---
 kernel/arch/x86-pc/linker.ld | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/kernel/arch/x86-pc/linker.ld b/kernel/arch/x86-pc/linker.ld
index 1e53287..aeb32e2 100644
--- a/kernel/arch/x86-pc/linker.ld
+++ b/kernel/arch/x86-pc/linker.ld
@@ -43,6 +43,13 @@ SECTIONS
         ebss = .;
     }
 
+    /* Define sections to fulfill unresolved references */
+    .unresolved :
+    {
+        /* Define symbols to fulfill unresolved references */
+        PROVIDE (_dummy_symbol = 0); /* Replace _dummy_symbol with the actual symbol name */
+    }
+
     /* End of the kernel */
     __kernel_end = .;
 }
-- 
2.43.0

