From e23eb78fcef06c0ef84c6b94012f2cfd3a5c5b9e Mon Sep 17 00:00:00 2001
From: Heisenberg <alimirmohammad.1386@gmail.com>
Date: Sat, 6 Apr 2024 15:54:19 -0400
Subject: [PATCH 388/493] Patch

---
 Makefile | 10 ++++++++--
 1 file changed, 8 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index 459c793..61f474c 100644
--- a/Makefile
+++ b/Makefile
@@ -1,4 +1,3 @@
-# The makefile for the CactusOS project, this file will get more complicated when the OS is in a later stage of development.
 
 ##########
 # .s files are GAS assembly
@@ -23,6 +22,10 @@
 # pci-ohci
 #######################
 
+VERSION = 1
+PATCHLEVEL = 4
+SUBLEVEL = 0
+
 INCLUDEDIRS := kernelz/include
 INCLUDEZ = libz/include
 QEMUOPTIONS := -boot d -device VGA,edid=on,xres=1024,yres=768 -trace events=../qemuTrace.txt -d cpu_reset #-readconfig qemu-usb-config.cfg -drive if=none,id=stick,file=disk.img -device usb-storage,bus=ehci.0,drive=stick
@@ -159,4 +162,7 @@ filelist:
 	@echo "Source Files:"
 	@echo -$(KRNLFILES)
 	@echo "Object Files:"
-	@echo -$(KRNLOBJS)
\ No newline at end of file
+	@echo -$(KRNLOBJS)
+
+version:
+	@echo $(VERSION)
\ No newline at end of file
-- 
2.43.0

