From 6d5a241f1450f667239fa70a55dc88c6ecb3d977 Mon Sep 17 00:00:00 2001
From: Heisenberg <alimirmohammad.1386@gmail.com>
Date: Sat, 6 Apr 2024 21:49:58 -0400
Subject: [PATCH 392/493] Minor patch

---
 .gitignore |  3 ++-
 Makefile   | 33 ++++++++++++++++++++++++++++++++-
 2 files changed, 34 insertions(+), 2 deletions(-)

diff --git a/.gitignore b/.gitignore
index b14ad95..0b617d4 100644
--- a/.gitignore
+++ b/.gitignore
@@ -2,4 +2,5 @@
 *.iso
 *.bin
 *.o
-*.sym
\ No newline at end of file
+*.sym
+*.config
\ No newline at end of file
diff --git a/Makefile b/Makefile
index f63d703..51a45f1 100644
--- a/Makefile
+++ b/Makefile
@@ -27,6 +27,16 @@ PATCHLEVEL = 4
 SUBLEVEL = 0
 EXTRAVERSION = -rc2
 
+CONFIG_FILE := .config
+
+ifeq ($(filter undefine,$(.FEATURES)),)
+    ifeq ($(shell expr $(MAKE_VERSION) \>= 3.80), 0)
+        $(error GNU Make >= 3.80 is required. Your Make version is $(MAKE_VERSION))
+    endif
+endif
+
+
+
 INCLUDEDIRS := kernelz/include
 INCLUDEZ = libz/include
 QEMUOPTIONS := -boot d -device VGA,edid=on,xres=1024,yres=768 -trace events=../qemuTrace.txt -d cpu_reset #-readconfig qemu-usb-config.cfg -drive if=none,id=stick,file=disk.img -device usb-storage,bus=ehci.0,drive=stick
@@ -176,4 +186,25 @@ filelist:
 	@echo -$(KRNLOBJS)
 
 version:
-	@echo $(KERNELVERSION)
\ No newline at end of file
+	@echo $(KERNELVERSION)
+# dialogconfig:
+# 	dialog --no-shadow --backtitle "Kernel Configuration" --title "Heisen Kernel Configuration" --clear --stdout --checklist "Select features to enable:" 20 60 10 \
+# 	    1 "General setup" on \
+# 	    2 "EHCI" off \
+# 	    3 "UHCI" off \
+# 	    4 "OHCI" off \
+# 	    > .config; \
+# 	if grep -q "1" .config; then \
+# 	    dialog --no-shadow --backtitle "Basic setup" --msgbox "Basic setup Config" 10 40; \
+# 			1 "Compile Heisen Kernel with GCC cross compiler" on \
+# 			> .config; \
+# 	fi || true
+
+
+
+# # Define the target to build the kernel
+# build:
+# 	# Placeholder for actual build commands
+# 	@echo "Building kernel..."
+
+
-- 
2.43.0

