From d32705e8965266da4c3b86a284381bb97b890b93 Mon Sep 17 00:00:00 2001
From: Heisenberg <alimirmohammad.1386@gmail.com>
Date: Sun, 7 Apr 2024 17:28:52 -0400
Subject: [PATCH 402/493] Update

---
 Makefile | 79 ++++++++++----------------------------------------------
 1 file changed, 14 insertions(+), 65 deletions(-)

diff --git a/Makefile b/Makefile
index c9ddd93..8cf99e9 100644
--- a/Makefile
+++ b/Makefile
@@ -1,4 +1,5 @@
-# Makefile for the HeisenKernel project under the MIT 
+# The makefile for the CactusOS project, this file will get more complicated when the OS is in a later stage of development.
+
 ##########
 # .s files are GAS assembly
 # .asm files are nasm assembly
@@ -22,29 +23,11 @@
 # pci-ohci
 #######################
 
-VERSION = 1
-PATCHLEVEL = 4
-SUBLEVEL = 0
-EXTRAVERSION = -rc2
-
-CONFIG_FILE := .config
-
-ifeq ($(filter undefine,$(.FEATURES)),)
-    ifeq ($(shell expr $(MAKE_VERSION) \>= 3.80), 0)
-        $(error GNU Make >= 3.80 is required. Your Make version is $(MAKE_VERSION))
-    endif
-endif
-
-
-
 INCLUDEDIRS := kernelz/include
-INCLUDEZ = libz/include
 QEMUOPTIONS := -boot d -device VGA,edid=on,xres=1024,yres=768 -trace events=../qemuTrace.txt -d cpu_reset #-readconfig qemu-usb-config.cfg -drive if=none,id=stick,file=disk.img -device usb-storage,bus=ehci.0,drive=stick
-ARCHINCLUDE := kernel/include
-
 
-G++PARAMS := -m32 -g -D CACTUSOSKERNEL -I $(INCLUDEDIRS) -I $(INCLUDEZ) -I $(ARCHINCLUDE)  -Wall -fno-omit-frame-pointer -fno-use-cxa-atexit -nostdlib -fno-builtin -fno-exceptions -fno-rtti -fno-leading-underscore -Wno-write-strings -fpermissive -Wno-unknown-pragmas 
-GCCPARAMS := -m32 -g -D CACTUSOSKERNEL -I $(INCLUDEDIRS) -I $(INCLUDEZ) -I $(ARCHINCLUDE) -Wall -fno-omit-frame-pointer -nostdlib -fno-builtin -Wno-unknown-pragmas
+G++PARAMS := -m32 -g -D CACTUSOSKERNEL -I $(INCLUDEDIRS) -Wall -fno-omit-frame-pointer -fno-use-cxa-atexit -nostdlib -fno-builtin -fno-exceptions -fno-rtti -fno-leading-underscore -Wno-write-strings -fpermissive -Wno-unknown-pragmas
+GCCPARAMS := -m32 -g -D CACTUSOSKERNEL -I $(INCLUDEDIRS) -Wall -fno-omit-frame-pointer -nostdlib -fno-builtin -Wno-unknown-pragmas
 ASPARAMS := --32
 LDPARAMS := -m elf_i386
 
@@ -62,9 +45,6 @@ KRNLOBJS := $(subst $(KRNLSRCDIR),$(KRNLOBJDIR),$(KRNLOBJS)) #Replace the kernel
 $(KRNLOBJDIR)/%.o: $(KRNLSRCDIR)/%.cpp
 	mkdir -p $(@D)
 	i686-elf-g++ $(G++PARAMS) -c -o $@ $<
-$(KRNLOBJDIR)/%.o: $(KRNLSRCDIR)/%.cc
-	mkdir -p $(@D)
-	i686-elf-g++ $(G++PARAMS) -c -o $@ $<
 
 ####################################
 #C source files
@@ -94,34 +74,27 @@ $(KRNLOBJDIR)/%.o: $(KRNLSRCDIR)/%.asm
 	mkdir -p $(@D)
 	nasm -f elf32 -O0 $< -o $@
 
-####################################
-#State the kernel version
-KERNELVERSION := $(VERSION)$(if $(PATCHLEVEL),.$(PATCHLEVEL)$(if $(SUBLEVEL),.$(SUBLEVEL)))$(EXTRAVERSION)
-export VERSION PATCHLEVEL SUBLEVEL KERNELRELEASE KERNELVERSION
-####################################
-
 
 
-HeisenOs.bin: kernelz/linker.ld $(KRNLOBJS)
+CactusOS.bin: kernelz/linker.ld $(KRNLOBJS)
 	i686-elf-ld $(LDPARAMS) -T $< -o $@ $(KRNLOBJS)
 
-CactusOS.iso: HeisenOs.bin
+CactusOS.iso: CactusOS.bin
 	cd lib/ && $(MAKE)
 	cd apps/ && $(MAKE)
-	cd drivers/ && $(MAKE)
 	
-	nm -a HeisenOs.bin | sort -d > isofiles/debug.sym
+	nm -a CactusOS.bin | sort -d > isofiles/debug.sym
 	cp -r isofiles/. iso
 	mkdir iso/boot
 	mkdir iso/boot/grub
-	cp HeisenOs.bin iso/boot/HeisenOs.bin
+	cp CactusOS.bin iso/boot/CactusOS.bin
 	cp grub.cfg iso/boot/grub/grub.cfg
 	grub-mkrescue --output=CactusOS.iso iso
 	rm -rf iso
 
 .PHONY: clean qemu kdbg run filelist serialDBG qemuDBG fastApps
 clean:
-	rm -rf $(KRNLOBJDIR) HeisenOs.bin CactusOS.iso
+	rm -rf $(KRNLOBJDIR) CactusOS.bin CactusOS.iso
 	cd lib/ && $(MAKE) clean
 	cd apps/ && $(MAKE) clean
 	rm -rf isofiles/apps/*.bin
@@ -135,7 +108,7 @@ qemuDBG: CactusOS.iso
 
 qemuGDB: CactusOS.iso
 	qemu-system-i386 -cdrom CactusOS.iso $(QEMUOPTIONS) -serial pty &
-	gdb -ex 'file HeisenOs.bin' -ex 'target remote /dev/pts/1' -q
+	gdb -ex 'file CactusOS.bin' -ex 'target remote /dev/pts/1' -q
 
 run: CactusOS.iso
 	vboxmanage startvm "CactusOS" -E VBOX_GUI_DBG_AUTO_SHOW=true -E VBOX_GUI_DBG_ENABLED=true &
@@ -149,7 +122,7 @@ serialDBG:
 
 kdbg: CactusOS.iso
 	qemu-system-i386 $(QEMUOPTIONS) -cdrom CactusOS.iso -serial stdio -s -S &
-	kdbg -r localhost:1234 HeisenOs.bin
+	kdbg -r localhost:1234 CactusOS.bin
 
 grub-core:
 	grub-mkimage -o isofiles/setup/core.img -O i386-pc -p="(hd0,msdos1)/boot/grub" --config=grubcore.cfg -v configfile biosdisk part_msdos fat normal multiboot echo
@@ -166,11 +139,11 @@ turboApps:
 	cd apps/ && $(MAKE) clean && $(MAKE)
 	rm CactusOS.iso
 
-installUSB: CactusOS.iso HeisenOs.bin isofiles/debug.sym isofiles/apps
+installUSB: CactusOS.iso CactusOS.bin isofiles/debug.sym isofiles/apps
 	rm -rf /media/remco/ISOIMAGE/apps/*.bin
 	cp -r isofiles/apps/* /media/remco/ISOIMAGE/apps/
 	cp isofiles/debug.sym /media/remco/ISOIMAGE/debug.sym
-	cp HeisenOs.bin /media/remco/ISOIMAGE/boot/HeisenOs.bin
+	cp CactusOS.bin /media/remco/ISOIMAGE/boot/CactusOS.bin
 	umount /media/remco/ISOIMAGE
 
 debug: CactusOS.iso
@@ -183,28 +156,4 @@ filelist:
 	@echo "Source Files:"
 	@echo -$(KRNLFILES)
 	@echo "Object Files:"
-	@echo -$(KRNLOBJS)
-
-version:
-	@echo $(KERNELVERSION)
-dialogconfig:
-	dialog --no-shadow --backtitle "Kernel Configuration" --title "Heisen Kernel Configuration" --clear --stdout --checklist "Select features to enable:" 20 60 10 \
-	    1 "General setup" on \
-	    2 "EHCI" off \
-	    3 "UHCI" off \
-	    4 "OHCI" off \
-	    > .config; \
-	if grep -q "1" .config; then \
-	    dialog --no-shadow --backtitle "Basic setup" --msgbox "Basic setup Config" 10 40; \
-			1 "Compile Heisen Kernel with GCC cross compiler" on \
-			> .config; \
-	fi || true
-
-
-
-# Define the target to build the kernel
-build:
-	# Placeholder for actual build commands
-	@echo "Building kernel..."
-
-
+	@echo -$(KRNLOBJS)
\ No newline at end of file
-- 
2.43.0

