From 534939fa6a5ef3290852b1ce4230ee0c147d5b48 Mon Sep 17 00:00:00 2001
From: Heisenberg <alimirmohammad.1386@gmail.com>
Date: Tue, 9 Apr 2024 18:53:16 -0400
Subject: [PATCH 442/493] Update and some patches

---
 drivers/usb3/driver_ftdi.c | 21 +++++++++++++++++++++
 1 file changed, 21 insertions(+)

diff --git a/drivers/usb3/driver_ftdi.c b/drivers/usb3/driver_ftdi.c
index 7ee778c..22b5730 100644
--- a/drivers/usb3/driver_ftdi.c
+++ b/drivers/usb3/driver_ftdi.c
@@ -173,4 +173,25 @@ int ftdi_check_supported_by_vid_pid(uint16_t vid,uint16_t pid){
     default:
         return 0;
     }
+}
+
+static const char* ftdi_get_device_name(uint16_t vid,uint16_t pid,uint8_t classes,uint8_t subclasses){
+    assert(FTDI_VENDOR_ID == vid);
+        switch (pid)
+    {
+    case FTDI_PRODUCT_ID_FT232R:
+    case FTDI_PRODUCT_ID_FT232R_FAKE: return FTDI_DEVICE_NAME_FT232R;
+    case FTDI_PRODUCT_ID_FT2232: return FTDI_DEVICE_NAME_FT2232;
+    case FTDI_PRODUCT_ID_FT4232H: return FTDI_DEVICE_NAME_FT4232H;
+    case FTDI_PRODUCT_ID_FT232H: return FTDI_DEVICE_NAME_FT232H;        
+    case FTDI_PRODUCT_ID_FT231X: return FTDI_DEVICE_NAME_FT231X;
+    case FTDI_PRODUCT_ID_FT232RL: return FTDI_DEVICE_NAME_FT232RL;
+    case FTDI_PRODUCT_ID_STK500: return FTDI_DEVICE_NAME_STK500;
+    case FTDI_PRODUCT_ID_OPENMOKO: return FTDI_DEVICE_NAME_OPENMOKO;
+    case FTDI_PRODUCT_ID_TUMPA: return FTDI_DEVICE_NAME_TUMPA;
+    case FTDI_PRODUCT_ID_KTLINK: return FTDI_DEVICE_NAME_KTLINK;
+    case FTDI_PRODUCT_ID_JTAGKEY: return FTDI_DEVICE_NAME_JTAGKEY;
+
+    default: return FTDI_DEVICE_NAME_GENERIC;
+    }
 }
\ No newline at end of file
-- 
2.43.0

