From ce7e68fe7546e395cec86afd7f13f3fdc83ca68d Mon Sep 17 00:00:00 2001
From: Heisenberg <alimirmohammad.1386@gmail.com>
Date: Wed, 10 Apr 2024 21:32:40 -0400
Subject: [PATCH 460/493] Patch

---
 apps/welcome/welcome.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/apps/welcome/welcome.cpp b/apps/welcome/welcome.cpp
index 6078cff..52a1ba3 100644
--- a/apps/welcome/welcome.cpp
+++ b/apps/welcome/welcome.cpp
@@ -19,8 +19,8 @@ const int numOfMines = 16;
 
 static const char multi_line_string[] = 
 "Heisen Os , licensed under MIT license, written by Heisenberg and intended to provide utility for personal use\n"
-"\nIt is recommended to reboot your system after installing Os."
-"\n";
+"\nIt is recommended to reboot your system after installing Os.\n"
+"\nOr you can just continue using it.\n";
 
 
 
-- 
2.43.0

