From 61e26f874ecb9138a6191f7c015aa4202135bbd6 Mon Sep 17 00:00:00 2001
From: Heisenberg <alimirmohammad.1386@gmail.com>
Date: Thu, 11 Apr 2024 17:56:50 -0400
Subject: [PATCH 471/493] Another update

---
 apps/chess/square.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/apps/chess/square.h b/apps/chess/square.h
index 73af7bf..7115d4d 100644
--- a/apps/chess/square.h
+++ b/apps/chess/square.h
@@ -15,6 +15,8 @@ namespace LIBHeisenKernel{
         const int squareNb = FileNb*RandNb;
 
         const unsigned int FileInc = 1;
+
+        const unsigned int RankInc = 16;
     };
 };
 
-- 
2.43.0

