From c7a235cd04bb9679d804bec94284bd4d6f3da669 Mon Sep 17 00:00:00 2001
From: Heisenberg <alimirmohammad.1386@gmail.com>
Date: Thu, 11 Apr 2024 18:00:17 -0400
Subject: [PATCH 473/493] Another update

---
 apps/chess/square.h | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/apps/chess/square.h b/apps/chess/square.h
index a7e790e..53b29e5 100644
--- a/apps/chess/square.h
+++ b/apps/chess/square.h
@@ -30,6 +30,17 @@ namespace LIBHeisenKernel{
         const int FileH = 0x0B;
 
         const int RankNone = 0;
+
+        const int Rank1 = 0x4;
+        
+        const int Rank2 = 0x5;
+        const int Rank3 = 0x6;
+        const int Rank4 = 0x7;
+        const int Rank5 = 0x8;
+        const int Rank6 = 0x9;
+        const int Rank7 = 0xA;
+        const int Rank8 = 0xB;
+
     };
 };
 
-- 
2.43.0

